decidim-core/app/views/decidim/widgets/show.js.erb

```diff
@@ -0,0 +1,3 @@
+document.write('<link type="text/css" rel="stylesheet" media="all" href="<%= asset_url 'decidim/widget.css' %>"></link>');
```

Comment:
I think we already tried this but it wasn't working? Or at least the problem was that people had to include the JS function in their code. Maybe we can make people embed a JS that creates the iframe and the JS function? (if we go the iframe way)